### PR TITLE
Adding http:// prefix to the swagger WebServicesURL to prevent basePath validation errors

### PR DESCRIPTION
**Pull request by nikhiljindal**
For fixing https://github.com/GoogleCloudPlatform/kubernetes/issues/4083

### CODE REVIEW

**Comment by vmarmol** [comment]
Looks like it needs a rebase btw

File: api/swagger-spec/api.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "swaggerVersion": "1.2",
   "apiVersion": "",
-  "basePath": "127.0.0.1:6443",
+  "basePath": "http://127.0.0.1:6443",
```
Comment (bgrant0607) [inline api/swagger-spec/api.json]: Shouldn't this be https?

Comment (nikhiljindal) [inline api/swagger-spec/api.json]: Changed as discussed.

**Comment by nikhiljindal** [comment]
Rebased.

**Comment by vmarmol** [comment]
LGTM, can wait for @bgrant0607 if you'd like.

**Comment by bgrant0607** [comment]
LGTM. 

Any idea why the swagger output changed so drastically?

**Comment by nikhiljindal** [comment]
github is not allowing me to see the diffs in v1beta{1,2,3}.json.
The diff in resourceListing.json is just rearranging paths.
Maybe the diff in v1beta files is also just rearranging stuff.

We need to pull in
https://github.com/emicklei/go-restful/commit/358e2483dac774992c858945b509f560547c07ba so that we stop seeing the spurious rearrange diffs.

**Comment by satnam6502** [comment]
OK to merge?

**Comment by nikhiljindal** [comment]
Yes. Ready to merge.

**Pull request merged**